File: backend/src/main/java/pl/michalkruk/cinema/core/movie/MovieForm.java
package pl.michalkruk.cinema.core.movie;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
class MovieForm {
    private String title;
    private Genre genre;
    private AgeLimit ageLimit;
    private short duration;
    private short releaseYear;
    private Country country;
    private String description;
}

File: frontend/src/main/web/src/app/movie/movie-form.ts
export class MovieForm {
  constructor(
    public title: string,
    public genre: string,
    public ageLimit: string,
    public duration: number,
    public releaseYear: number,
    public country: string,
    public description: string,
    public image: File,
  ) {

  }
}
